apps/client/pages/register.tsx

```diff
@@ -110,17 +110,16 @@ export default function RegisterPage() {
                                 </div>
                             ) : null}
 
-                            <AuthDevTools isAdmin={isAdmin} setIsAdmin={setIsAdmin} />
```

Comment:
why was this removed?

Reply:
danngggittt. i totally missed that when i looked at the code commits. can't speak to why it was removed, but 100% on me for not catching.

Reply:
my bad! PR with restoring it coming

Reply:
all good, we can also use the fullWidth prop on the button rather than adding the class 👍

Reply:
follow up PR - https://github.com/maybe-finance/maybe/pull/208

sorry folks! regarding full width prop, good point. I find it a little bit weird to expose it that way but that's a topic for another time :)